src/Sylius/Bundle/ChannelBundle/spec/Sylius/Bundle/ChannelBundle/Form/Type/ChannelTypeSpec.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\ChannelBundle\Form\Type;
```

Comment:
Folder structure is wrong here, it shouldn't be `spec/Sylius/Bundle/ChannelBundle/Form/` but just `spec/Form/`, i.e. https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/AddressingBundle/spec/Form/Type/AddressTypeSpec.php#L12